packages/hdwallet-trezor/src/trezor.ts

```diff
@@ -258,28 +248,11 @@ export class TrezorHDWalletInfo implements core.HDWalletInfo, core.BTCWalletInfo
 export class TrezorHDWallet implements core.HDWallet, core.BTCWallet, core.ETHWallet {
   readonly _supportsETHInfo = true;
   readonly _supportsBTCInfo = true;
-  readonly _supportsDebugLink = false;
   readonly _supportsBTC = true;
   readonly _supportsETH = true;
-  readonly _supportsCosmosInfo = false;
-  readonly _supportsCosmos = false;
-  readonly _supportsBinanceInfo = false;
-  readonly _supportsBinance = false;
-  readonly _isTrezor = true;
-  readonly _supportsRippleInfo = false;
-  readonly _supportsRipple = false;
-  readonly _supportsEosInfo = false;
-  readonly _supportsEos = false;
-  readonly _supportsFioInfo = false;
-  readonly _supportsFio = false;
-  readonly _supportsThorchainInfo = false;
-  readonly _supportsThorchain = false;
-  readonly _supportsSecretInfo = false;
-  readonly _supportsSecret = false;
-  readonly _supportsKava = false;
   readonly _supportsKavaInfo = true;
-  readonly _supportsTerra = false;
   readonly _supportsTerraInfo = true;
+  readonly _isTrezor = true;
```

Comment:
Are you sure this isn't going to break anything? I would not be surprised in the least if there's code that directly references these instead of using the helper functions.

Reply:
Code that references these will still get `undefined` -- things will only break if something's doing `=== false` explicitly.